cmake/BLTMacros.cmake

```diff
@@ -5,6 +5,38 @@
 
 include(${BLT_ROOT_DIR}/cmake/BLTPrivateMacros.cmake)
 
+##------------------------------------------------------------------------------
+## blt_assert_exists( [DIRECTORY <path-to-dir>]
+##                    [TARGET <target>]
+##                    [FILE <path-to-file>] )
+##
+## Throws a FATAL_ERROR message if the specified directory, file, or target does
+## not exist.
+##------------------------------------------------------------------------------
+macro(blt_assert_exists)
+
+    set(options)
+    set(singleValueArgs DIRECTORY TARGET FILE EXECUTABLE )
```

Comment:
This is a nice feature @gzagaris.

Is ``EXECUTABLE`` supposed to be here? It is not described in the documentation or handled in the code.